src/mon/LogMonitor.h

```diff
@@ -206,6 +206,8 @@ class LogMonitor : public PaxosService,
       "mon_cluster_log_to_graylog",
       "mon_cluster_log_to_graylog_host",
       "mon_cluster_log_to_graylog_port",
+      "fsid",
```

Comment:
> so LogMonitor::handle_conf_change() also reads from conf.host and pass the host read from conf to graylog.

@xiexingguo i cannot find the line where `host` or `fsid` is passed down to graylog in `LogMonitor`, could you point me to it? thanks.